msg/tools/generate_microRTPS_bridge.py

```diff
@@ -78,91 +80,58 @@ def check_rtps_id_uniqueness(classifier):
     """
 
     repeated_ids = dict()
-
-    if sys.version_info[0] < 3:
-        full_send_list = dict(
-            list(
-                msg for msg in classifier.msgs_to_send.items()) +
-            list(
-                msg[0].items()[0] for msg in classifier.alias_msgs_to_send))
-        full_receive_list = dict(
-            list(
-                msg for msg in classifier.msgs_to_receive.items()) +
-            list(
-                msg[0].items()[0] for msg in classifier.alias_msgs_to_receive))
-        full_ignore_list = dict(
-            list(
-                msg for msg in classifier.msgs_to_ignore.items()) +
-            list(
-                msg[0].items()[0] for msg in classifier.alias_msgs_to_ignore))
-    else:
-        full_send_list = dict(
-            list(
-                msg for msg in classifier.msgs_to_send.items()) +
-            list(
-                list(
-                    msg[0].items())[0] for msg in classifier.alias_msgs_to_send))
-        full_receive_list = dict(
-            list(
-                msg for msg in classifier.msgs_to_receive.items()) +
-            list(
-                list(
-                    msg[0].items())[0] for msg in classifier.alias_msgs_to_receive))
-        full_ignore_list = dict(
-            list(
-                msg for msg in classifier.msgs_to_ignore.items()) +
-            list(
-                list(
-                    msg[0].items())[0] for msg in classifier.alias_msgs_to_ignore))
+    full_send_list = dict(
```

Comment:
Can you find a nicer way to write that? It looks less readable now.

Reply:
Yes I can. I just let `autopep8` do it's stuff.